docker-compose.yml

```diff
@@ -5,7 +5,9 @@ services:
         build: 
             context: ./twitch-chat-reader
         networks: 
-            - stream-net
+            stream-net:
+                aliases:
+                    - twitchreader
```

Comment:
you can totally change these to something you think is more appropriate, but this got it working at least